src/misc/image.v1.yaml

```diff
@@ -8,7 +8,7 @@ allOf:
             title: IIIF Image
             type: string
             format: uri
-            pattern: ^https://
+            pattern: ^https://[^/]+(/[^?#\[\]@]+)?/[^/?#\[\]@]+$ # http://iiif.io/api/image/2.1/#uri-encoding-and-decoding, with support for percent encoding
```

Comment:
Doesn't test for a valid authority. Assuming `format: uri` covers that.

Reply:
The first subgroup looks like an optional prefix like the IIIF server location, but then it seems to match the old URLs like `https://iiif.elifesciences.org/lax:09560/elife-09560-fig1-v1.tif` https://regex101.com/r/NngB53/3

Reply:
Yep, `lax:09560` is a valid prefix and `elife-09560-fig1-v1.tif` a valid identifier. (This is why slashes in identifiers needs to be escaped.)

Reply:
(also `https://iiif.elifesciences.org/lax/09560/elife-09560-fig1-v1.tif` as the end of the IIIF server path is indistuinguishable from the start of the IIIF file path)